TestComponentCSharp/ManualProjectionTestClasses.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include "IBindableIteratorTest.g.h"
+#include "IDictionaryTest.g.h"
+#include "IDisposableTest.g.h"
+
+namespace winrt::TestComponentCSharp::implementation
+{
+	struct IBindableIteratorTest : IBindableIteratorTestT<IBindableIteratorTest>
+	{
+		IBindableIteratorTest();
+		bool MoveNext();
+		Windows::Foundation::IInspectable Current();
+		bool HasCurrent();
+	};
+
+	struct IDictionaryTest : IDictionaryTestT<IDictionaryTest>
+	{
+	private:
+		Windows::Foundation::Collections::IMap<hstring, hstring> _map{
+			winrt::single_threaded_map<hstring, hstring>()
+		};
+	public:
+		IDictionaryTest();
+		hstring Lookup(hstring key);
+		bool HasKey(hstring key);
+		Windows::Foundation::Collections::IMapView<hstring, hstring> GetView();
+		bool Insert(hstring key, hstring value);
+		void Remove(hstring key);
+		void Clear();
+		int Size();
+		Windows::Foundation::Collections::IIterator<Windows::Foundation::Collections::IKeyValuePair<hstring, hstring>> First();
+
+		static bool Consume(Windows::Foundation::Collections::IMap<hstring, hstring> map);
+	};
+
+	struct IDisposableTest : IDisposableTestT<IDisposableTest>
+	{
+		IDisposableTest();
+		void Close();
+	};
+}
+
+namespace winrt::TestComponentCSharp::factory_implementation
+{
+	struct IBindableIteratorTest : IBindableIteratorTestT<IBindableIteratorTest, implementation::IBindableIteratorTest, Windows::Foundation::IStringable>
+	{
+		hstring ToString()
```

Comment:
looks like all these ToString overrides were copied from ComImports (same for NonAgileClass)?  They should be removed, since we're not testing them.